tests/ballerina-unit-test/src/test/java/org/ballerinalang/test/taintchecking/TaintedStatusPropagationTest.java

```diff
@@ -280,6 +280,22 @@ public void testIterableNegative() {
         BAssertUtil.validateError(result, 1, "tainted value passed to sensitive parameter 'secureIn'", 5, 53);
     }
 
+    @Test
+    public void testIterableWitinIterable() {
+        CompileResult result = BCompileUtil
+                .compile("test-src/taintchecking/propagation/iterable-within-iterable.bal");
```

Comment:
Shall we reformat this to a single line?

Reply:
+1. Will  also include other locations within this class.